pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -34,7 +34,7 @@ self: super: {
   jailbreak-cabal = (disableSharedExecutables super.jailbreak-cabal).override { Cabal = self.Cabal_1_20_0_4; };
 
   # enable using a local hoogle with extra packagages in the database
-  # nix-shell -p "haskellPackages.hoogleLocal (with haskellPackages; [ mtl lens ])"
+  # nix-shell -p "haskellPackages.hoogleLocal { packages = with haskellPackages; [ mtl lens ]; }"
```

Comment:
Though I can’t reconstruct right now why you have to use `overrideCabal` w/o default argument from the top of my head.